install/kubernetes/helm/istio-remote/templates/sidecar-injector-configmap.yaml

```diff
@@ -130,7 +132,6 @@ data:
           value: {{ "[[ or (index .ObjectMeta.Annotations \"sidecar.istio.io/interceptionMode\") .ProxyConfig.InterceptionMode.String ]]" }}
         imagePullPolicy: {{ .Values.global.imagePullPolicy }}
         securityContext:
-          privileged: false
```

Comment:
Why did you remove this ?

Reply:
By default it is false, we do not need to set it.

Reply:
There are other things that can be changed in the securityContext, and we do not add in the yaml the default value. This should be a best practice to keep the yaml files small.